Makefile

```diff
@@ -78,17 +81,17 @@ PLATFORMS ?= linux/arm64,linux/amd64
 docker-push: ## Build a multi-architecture image and push that into the registry
 	docker run --rm --privileged tonistiigi/binfmt:qemu-v6.2.0-26@sha256:5bf63a53ad6222538112b5ced0f1afb8509132773ea6dd3991a197464962854e --install all
 	docker version
-	docker buildx version
-	docker buildx create --use --name=image-builder 2>/dev/null || docker buildx use --default image-builder
-	# copy existing Dockerfile and insert --platform=${BUILDPLATFORM} into Dockerfile.cross, and preserve the original Dockerfile
-	sed -e '1 s/\(^FROM\)/FROM --platform=\$$\{BUILDPLATFORM\}/; t' -e ' 1,// s//FROM --platform=\$$\{BUILDPLATFORM\}/' Dockerfile > Dockerfile.cross
-	docker buildx build \
+	$(DOCKER_BUILDX) version
```

Comment:
Note that we override `DOCKER_BUILDX` in cloudbuild.yaml to exact location on the `buildx` binary. When someone runs this on their box, it defaults to `docker buildx`.